language/libra-tools/writeset-transaction-generator/templates/halt_transactions.move

```diff
@@ -0,0 +1,8 @@
+script {
+    use 0x1::LibraTransactionPublishingOption;
+    fun main(libra_root: &signer) {
+        LibraTransactionPublishingOption::set_open_script(libra_root);
+        // Add a null hash to the script allow list to disable execution of all scripts sent to the validator.
+        LibraTransactionPublishingOption::add_to_script_allow_list(libra_root, x"0000000000000000000000000000000000000000000000000000000000000000");
```

Comment:
TIL! I had a look around in the Move code, but can't seem to find where this check is being performed, is it happening at the VM level and not at the Move level?

Reply:
It is invoked in the prologue by `is_script_allowed` function!